main.tf

```diff
@@ -8,90 +8,96 @@ moved {
 }
 
 resource "tls_private_key" "ssh" {
-  count     = var.admin_username == null ? 0 : 1
+  count = var.admin_username == null ? 0 : 1
+
```

Comment:
Why this blank line ?

Reply:
Hi, I'd like to enforce a strict style guide in these official modules in the future, one rule will be:

>The resource meta-arguments `provider`, `count`, and `for_each` should come at the beginning of the resource block. The meta-arguments `depends_on` and `lifecycle` should be placed at the end of the resource block. An empty line should separate the meta-arguments from the resource arguments.